site/content/exercises/en-US/imagicharm/level-1/index.mdx

```diff
@@ -0,0 +1,8 @@
+---
+title: ImagiCharm Level 1
+---
```

Comment:
```suggestion
category: ImagiCharm
level: 1
---
``` 
See if this helps with the index. If not, we can merge and I'll make a separate PR to fix it, then another with developer experience improvements.